components/src/molecules/select/Select.html

```diff
@@ -1,15 +1,14 @@
 <div class="select">
   <a-label
     :for="id"
-    :hidden="hiddenLabel"
-    :inline="inlineLabel"
     :class="['select__label', labelClass]"
   >
-    {{ label }}
-   </a-label>
+    <!-- @slot Label text -->
+    <slot />
+  </a-label>
   <div class="select__field">
     <select
-      :id:="id"
```

Comment:
we need support for `v-model`. Please consider if we should use scoped slots or there is any other options

Reply:
support for v-model added, you can see changing value in actions tab, it now works similar to vsf `BaseSelect`

Reply:
I'm not sure if it's better to show value in actions tab or directly under select component as we have in `swatch` component, what do you prefer?

Reply:
actions addon is ok.